table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GlueDatabase.java

```diff
@@ -316,26 +320,33 @@ public UdbTable getTable(String tableName) throws IOException {
 
       // Glue does not provide column statistic information
       List<ColumnStatisticsInfo> columnStatisticsData = new ArrayList<>();
+      Map<String, String> tableParameters = table.getParameters() == null
+          ? new HashMap<>() : table.getParameters();
 
       PartitionInfo partitionInfo = PartitionInfo.newBuilder()
+          // Database name is not required for glue table, use mGlueDbName
           .setDbName(mGlueDbName)
           .setTableName(tableName)
           .addAllDataCols(GlueUtils.toProto(table.getStorageDescriptor().getColumns()))
           .setStorage(GlueUtils.toProto(table.getStorageDescriptor(), pathTranslator))
-          .putAllParameters(table.getParameters())
+          .putAllParameters(tableParameters)
           .build();
 
       Layout layout = Layout.newBuilder()
           .setLayoutType(HiveLayout.TYPE)
           .setLayoutData(partitionInfo.toByteString())
           .build();
 
-      List<Column> partitionColumns = table.getPartitionKeys();
+      if (table.getPartitionKeys() == null) {
+        partitionColumns = new ArrayList<>();
```

Comment:
Instead of creating a new instance, we can use `Collections.emptyList()`

Reply:
Changed.